pylzy/lzy/api/v1/local/runtime.py

```diff
@@ -163,7 +163,8 @@ async def __exec_call(self, call: LzyCall):
 
                 exception = await self.__workflow.snapshot.get_data(call.exception_id)
                 if isinstance(exception, Result):
-                    raise exception.value
+                    exc = exception.value
+                    raise exc[1].with_traceback(exc[2])
```

Comment:
Either rename exc to exc_info (because every Python developer knows this name) or deconstruct this exc tuple into named vars. Better to do both,